system/services/updateManager/AppDatabaseMigrationService.cfc

```diff
@@ -26,6 +26,19 @@ component {
 
 		$SystemOutput( "Running DB Migration: [#arguments.migration#]..." );
 
+		if ( $getColdbox().handlerExists( "dbmigrations.#arguments.migration#.isEnabled" ) ) {
```

Comment:
This is logically in the wrong place I feel. This logic should be in the *_getInstalledMigrations()* call. i.e. if they have `isEnabled()` and it returns false, we should consider them not installed.